MediaContent/composer.json

```diff
@@ -5,7 +5,9 @@
         "php": "~7.1.3||~7.2.0||~7.3.0",
         "magento/framework": "*",
         "magento/module-media-content-api": "*",
-        "magento/module-adobe-stock-asset-api": "*"
+        "magento/module-media-gallery-api": "*",
+        "magento/module-media-gallery": "*",
+        "magento/module-cms": "*"
```

Comment:
cms and catalog should be soft dependencies for the implementation - please add them to suggest section of composer.json